metrics/mypy_high_water_mark

```diff
@@ -1 +1 @@
-92.3800
\ No newline at end of file
+92.3400
```

Comment:
Unfortunately mypy throws any line that logs the name of a database engine into the 'untyped' category, as sqlalchemy-stubs doesn't have that field in its stubs for some reason.